### ISSUE TITLE
More structure maybe?

### ISSUE THREAD

**Issue by hackistic** [thatsimonsguy/seldon-plan#1]
I feel this projects needs much better structure for contributing.  Custom labels and tags etc.  You can check my profile I am developer and seek to help.  Im also a certified ethical hacker.

**Comment by jas0nmjames** [comment]
Agreed.  What about using Github projects?  There could be a simple kanban to track the different topics we'd like to cover.  I work in tech, would be happy to help as well.
